experimental/devices/pca9685/pca9685.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package pca9685
+
+import (
+	"time"
+
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/physic"
+)
+
+// I2CAddr i2c default address.
+const I2CAddr uint16 = 0x40
+
+// PCA9685 Commands
+const (
+	mode1      byte = 0x00
+	mode2      byte = 0x01
+	subAdr1    byte = 0x02
```

Comment:
These are bitmasks on mode1. I'd vote to remove them since they are not used and you are mixing register addresses and bitmasks here, which is super confusing when looking up the datasheet.

Well, there *are* subadr1~3 registers, but they are 0x19 to 01A.